tests/installation/bootloader.pm

```diff
@@ -54,7 +54,7 @@ sub run {
     push @params, bootmenu_network_source;
     push @params, bootmenu_remote_target;
     push @params, specific_bootmenu_params;
-    push @params, registration_bootloader_params(utils::VERY_SLOW_TYPING_SPEED);
+    push @params, registration_bootloader_params(utils::VERY_SLOW_TYPING_SPEED) unless get_var('SLP_RMT_INSTALL');
```

Comment:
Why need exclude it for 'SLP_RMT_INSTALL'?

Reply:
If we add https://scc.suse.com it will not pop up the screen for the
slp discovery.  http://10.161.8.44/tests/165#step/welcome/3

Reply:
If not skip it will add https://scc.suse.com and cause the installer to connect directly
 to scc.suse.com. It will not scan for the slp and will not pop up the screen for the
slp discovery.  http://10.161.8.44/tests/165#step/welcome/3